packages/product-components/src/components/PassphraseTypeCard/NonAsciiBanner.tsx

```diff
@@ -0,0 +1,79 @@
+import React, { ComponentProps } from 'react';
+import { FormattedMessage } from 'react-intl';
+import styled from 'styled-components';
+import { Banner, Button, Code, Text } from '@trezor/components';
+import { borders, spacingsPx, typography } from '@trezor/theme';
+
+const ButtonWrapper = styled.div`
+    flex-shrink: 0;
+    display: flex;
+    align-items: center;
+`;
+
+const SpecialCharsWrapper = styled.div`
+    display: flex;
+    flex-direction: column;
+    margin: ${spacingsPx.xxs} 0 0;
+    ${typography.label};
+    overflow: hidden;
+    border-radius: ${borders.radii.xs};
+`;
+
+const SpecialChars = styled.div`
+    padding: 0 ${spacingsPx.xxs};
+    box-shadow: inset 0 0 0 2px #f2f2f2;
+    font-family: RobotoMono, PixelOperatorMono8, monospace;
+    font-weight: 400;
+    letter-spacing: 0.1em;
+    background-color: ${({ theme }) => theme.backgroundNeutralSubtleOnElevation1};
+`;
+
+const SpecialDescription = styled.div`
+    background-color: ${({ theme }) => theme.backgroundNeutralDisabled};
+    padding: ${spacingsPx.xxs};
+    display: flex;
+    gap: ${spacingsPx.sm};
+    ${typography.label}
+`;
+
+type Props = {
+    variant: ComponentProps<typeof Banner>['variant'];
+};
+
+export const NonAsciiBanner = ({ variant }: Props) => {
+    return (
+        <Banner variant={variant} color="default" spacingX="xs">
+            <Text>
+                <FormattedMessage
+                    defaultMessage="We recommend using <code>ABC</code>, <code>abc</code>, <code>123</code>, <code>spaces</code> or <code>these special characters</code>"
+                    id="TR_PASSPHRASE_NON_ASCII_CHARS"
+                    values={{ code: text => <Code>{text}</Code> }}
+                />
+            </Text>
+            <SpecialCharsWrapper>
+                <SpecialChars>
+                    {'! " # $ % & \\ \' ( ) * +  - . / : ; < = > ? @ [  ] ^ _ ` { | } ~'}
+                </SpecialChars>
+                <SpecialDescription>
+                    <FormattedMessage
+                        defaultMessage="Using non-listed special characters risk future compatibility"
+                        id="TR_PASSPHRASE_NON_ASCII_CHARS_WARNING"
+                    />
+                    <ButtonWrapper>
+                        {/* TODO: better would be to reuse LearnMoreButton */}
```

Comment:
I noticed in suite, there is `LearnMoreButton`, but I didn't want to introduce circular dependency between packages, so I duplicated the code for now.